src/raygun.utilities.js

```diff
@@ -413,6 +413,10 @@ window.raygunUtilityFactory = function(window, Raygun) {
       return window.location.origin;
     },
     resolveFullUrl: function(url) {
+      if(url && url.indexOf('//') === 0) {
```

Comment:
What was this for?

Reply:
Urls in the format `//webapi.raygun.com/...`